m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2509,7 +2534,10 @@ PROCEDURE READCONFIGFILESETTINGS
   END
  CALL BNODEBUGMSGADD
 
- BNODEBUGMSG="DB:"+FORMAT("9",DEBUGMODE)
+ IF PARAMDEBUGMODE<>-1 THEN
+  BNODEBUGMSG="DB:"+PARAMDEBUGMODECHR
+ CALL BNODEBUGMSGADD
```

Comment:
Should you have a DO/END block with BNODEBUGMSG= and BNODEBUGMSGADD call.  You are only wanting to add that debug message if PARAMEDEBUGMODE<>-1, right.

Reply:
Thank you. Good catch! I've added that to the code, and it will be in the next commit.